### DISCUSSION TITLE
모바일 에러 질문드려요 ㅠㅜㅜ

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#809] [5. Q&A]
심사에서 리젝되서 건물과 맵들을 최적화를 진행했습니다.

 그 이후로 모바일로 테스트 했을때, 캐릭터가 계속 허공에서 허우적거립니다. 다른게 안보이고요. 

왜 그러는 걸까요? ㅜㅜ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 도움을 드리기 위해서는 개발 환경 정보, 스크린샷 또는 영상, 코드를 첨부해주시기를 부탁 드립니다.
